layers/+lang/rust/packages.el

```diff
@@ -72,3 +73,16 @@
       (spacemacs/declare-prefix-for-mode 'rust-mode "mg" "goto")
       (spacemacs/set-leader-keys-for-major-mode 'rust-mode
         "gg" 'racer-find-definition))))
+
+(defun rust/init-rustfmt ()
+  (use-package rustfmt
+    :defer t
+    :init
+    (progn
+      (if rust-enable-rustfmt-on-save
+          (spacemacs/add-to-hook 'rust-mode-hook
+                                 '(rustfmt-enable-on-save)))
+
+      (spacemacs/set-leader-keys-for-major-mode 'rust-mode
+        "=" 'rustfmt-format-buffer))
+    ))
```

Comment:
No, whitespace between trailing `)`. Format it like this: `"=" 'rustfmt-format-buffer))))`